airgun/widgets.py

```diff
@@ -819,17 +819,9 @@ class Search(Widget):
         'or contains(@class, "dataTables_filter") or @id="search-bar"]'
     )
     search_field = TextInput(
-        locator=(
-            ".//input[@id='search' or contains(@placeholder, 'Filter') or "
-            "@ng-model='table.searchTerm' or contains(@ng-model, 'Filter') or "
-            "@data-autocomplete-id='searchBar' or contains(@placeholder, 'Search') "
-            "or contains(@class, 'search-input')]"
-        )
-    )
-    search_button = Text(
-        ".//button[contains(@id, 'btn-search') or contains(@type,'submit') or "
-        "contains(@class, 'search-btn') or @ng-click='table.search(table.searchTerm)']"
+        locator=("//div[@class='title_filter']//input[contains(@aria-label, 'Search input')]")
     )
```

Comment:
I suggest keeping this as it is and adding this section to the locator **`[contains(@aria-label, 'Search input')]`**, as there may be cases where the search input has a different locator. For example, if you navigate to Hosts → All Hosts → Reports, this locator would not work in that context.
```suggestion
    search_field = TextInput(
        locator=(
            ".//input[@id='search' or contains(@placeholder, 'Filter') or "
            "@ng-model='table.searchTerm' or contains(@ng-model, 'Filter') or "
            "@data-autocomplete-id='searchBar' or contains(@placeholder, 'Search') "
            "or contains(@class, 'search-input') or contains(@aria-label, 'Search input')]"
        )
    )
```

Reply:
Also, the search field you referred has its own search_bar parameter.